test/recipes/20-test_pkeyutl.t

```diff
@@ -210,20 +210,21 @@ SKIP: {
         if disabled("rsa");
 
     # Self-compat
-    ok(run(app(([ 'openssl', 'pkeyutl', '-encap', '-pubin', '-kemop', 'RSASVE',
+    ok(run(app(([ 'openssl', 'pkeyutl', '-encap', '-kemop', 'RSASVE',
                   '-inkey', srctop_file('test', 'testrsa2048pub.pem'),
                   '-out', 'encap_out.bin', '-secret', 'secret.bin']))),
                   "RSA pubkey encapsulation");
     ok(run(app(([ 'openssl', 'pkeyutl', '-decap', '-kemop', 'RSASVE',
                   '-inkey', srctop_file('test', 'testrsa2048.pem'),
-                  '-in', 'encap_out.bin', '-out', 'decap_out.bin']))),
```

Comment:
I think that if we keep a backward compatibility, we also should keep the test for it

Reply:
Test added.